pep/app/constants/regex.ts

```diff
@@ -10,3 +10,4 @@ export const PARENTHESIS_REGEX = /\((.*)\)/;
 export const NON_DIGIT_REGEX = /\D/g;
 export const SEARCH_STRING_REGEX = /\(([^()]+)\)/g;
 export const SEARCH_STRING_TERMS_REGEX = /[^a-zA-Z\s-]+/g;
+export const URL_REGEX = /https?:\/\/(www\.)?[-a-zA-Z0-9@:%._\+~#=]{1,256}\.[a-zA-Z0-9()]{1,6}\b([-a-zA-Z0-9()@:%_\+.~#?&//=]*)/;
```

Comment:
ember-changeset-validations should already have `url` format validation built-in:

https://github.com/poteto/ember-changeset-validations#format

Reply:
They do - however they don't require the https or http which we do